pkgs/development/python-modules/pyqt/6.x.nix

```
@@ -105,7 +107,8 @@ buildPythonPackage rec {
     ++ lib.optional withMultimedia qtmultimedia
     ++ lib.optional withWebSockets qtwebsockets
     ++ lib.optional withLocation qtlocation
-    ++ lib.optional withSerialPort qtserialport;
+    ++ lib.optional withSerialPort qtserialport
+    ++ lib.optional withSpeech qt6.qtspeech;
```

qt6 prefix should be unneeded, you're in a `with qt6Packages;`


```suggestion
    ++ lib.optional withSpeech qtspeech;
```